camp/abolotski/XBoxToTalon/src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+/**
+ * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
+ * constants. This class should not be used for any other purpose. All constants should be declared
+ * globally (i.e. public static). Do not put anything functional in this class.
+ *
+ * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * constants are needed, to reduce verbosity.
+ */
+public final class Constants {
+    public static final int JoystickMotorCanId = 2;
+
+    public static final int PortMainController = 0;
```

Comment:
could you give this a better name? I'm not sure what it's for from this name.

Reply:
talked inperson